### DISCUSSION TITLE
Pop-up data query tabs just spin? (firefox, macos, *.servicenowservices.com instance)

### DISCUSSION THREAD

**Discussion by tezdog** [arnoudkooi/SN-Utils#232] [Q&A] [answered]
This must be something unusual in my environment, but when I open the pop-up menu from the icon in the URL bar, and try any of the first six tabs (Update set ... User) they just show a spinner on the tab rather than returning data.  For instance the User tab, I enter my own username, and click Go, and it seems to spin forever.  I am logged in as an admin, and even have done elevate rolls 'security_admin'.  If I go to the sys_user table, I can easily search and find my record.  Only 24k records in the table.  I am using a firefox browser, on a macos system, connected to a *.servicenowservices.com instance.

Any ideas what might be the problem?  Anyone with a similar environment have these working? 
Thanks.

**Comment by arnoudkooi** [comment, marked as answer]
Still seeing this issue? You do have admin role?

**Reply by tezdog** [reply]
Yes, still seeing it.  Logged in to account with admin role, even elevated to 'security_admin'.
SN Utils 6.0.3.0 on Firefox 91.9.0esr, with a *.servicenowservices.com domain, macOS 10.15.7 (Catalina) with latest updates.
I don't think these tabs have ever worked for me.  Thanks,  - Tim

**Reply by arnoudkooi** [reply]
I just did a minor update to the manifest file.
Can you update to 6.0.3.3 and try (It May need a browser restart)

**Reply by tezdog** [reply]
Yes!  You fixed it.  Thanks again for the fantastic extension.

**Reply by arnoudkooi** [reply]
Cool, it was only a missing wildcard character!
